### DISCUSSION TITLE
GUI showoff

### DISCUSSION THREAD

**Discussion by IshanJ25** [TomSchimansky/CustomTkinter#288] [Showcase]
Here is a screenshot of one of the pages in the app I am making using CTk. Custom color palette has been used.

![image](https://user-images.githubusercontent.com/86649457/180046656-b97a6640-332f-4bb8-a215-638ca9354c28.png)

(Icons from google fonts)

**Comment by felipetesc** [comment]
> Here is a screenshot of one of the pages in the app I am making using CTk. Custom color palette has been used.
> 
> ![image](https://user-images.githubusercontent.com/86649457/180046656-b97a6640-332f-4bb8-a215-638ca9354c28.png)
> 
> (Icons from google fonts)

Hey, this is nice,  right now I'm in need for a calendar widget and I'm using tkcalendar for that. Are you using the same lib?

**Reply by IshanJ25** [reply]
yes, but an alternative is to use 3 option menus together
(for Day, Month, Year)

It provides the basic functionality but is not as efficient as tkcalendar

**Comment by Naviamold1** [comment]
hey it its not a problem could you send me a calendar and add transaction style config?
